### PR TITLE
about dialog: fix small typo

### PR DESCRIPTION
**Pull request by HuanJiCanShang**

### CODE REVIEW

File: lang/zh-s.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
 		<vita3k_staff>Vita3K人员名单</vita3k_staff>
 		<developers>开发人员</developers>
 		<contributors>贡献者</contributors>
+		<contributors>支持者</contributors>
```
Comment (Zangetsu38) [inline lang/zh-s.xml:37]: fail

File: lang/zh-t.xml [hunk 2]
```diff
@@ -34,6 +34,7 @@
 		<vita3k_staff>Vita3K人員名單</vita3k_staff>
 		<developers>開發人員</developers>
 		<contributors>參與者</contributors>
+		<contributors>支持者</contributors>
```
Comment (Zangetsu38) [inline lang/zh-t.xml:37]: same

File: lang/zh-s.xml (hunk 1 above)
Comment (HuanJiCanShang) [reply lang/zh-s.xml]: changed

File: lang/zh-t.xml (hunk 2 above)
Comment (HuanJiCanShang) [reply lang/zh-t.xml]: changed

**Comment by Zangetsu38** [comment]
@Croden1999 🙏

**Comment by Croden1999** [comment]
> @Croden1999 🙏

Lgtm, Mr. @Zangetsu38 , it is fine

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**